README.md

```diff
@@ -213,26 +213,29 @@ Short backend is built on top of
 [Uncle Bob's Clean Architecture](https://api.short-d.com/r/ca), the central
 objective of which is separation of concerns.
 
-![Clean Architecture](doc/eng/clean-architecture/clean-architecture.jpg)
-![Boundary](doc/eng/clean-architecture/boundary.jpg)
+![Clean Architecture](doc/clean-architecture/clean-architecture.jpg)
```

Comment:
Do we need this image here? The image by itself doesn't provide much information without the explanation from the blog. We have our infrastructure in the next image for devs to refer who already understood the architecture pattern.

Reply:
@rohithbalaji123 People previously mentioned that they are interested in clean architecture when looking at the repo. So I kept this diagram here to attack developers.

Reply:
Okay, agreed!